vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -185,39 +180,47 @@ def parse_selector_parts(entity)
 
     def parse_container_definition(container_def, pod_id)
       new_result = {
-        :ems_ref           => "#{pod_id}_#{container_def["name"]}_#{container_def["image"]}",
-        :name              => container_def["name"],
-        :image             => container_def["image"],
-        :image_pull_policy => container_def["imagePullPolicy"],
-        :memory            => container_def["memory"],
+        :ems_ref           => "#{pod_id}_" \
+                              "#{container_def.name}_" \
```

Comment:
Please remove the line continuators...they are not needed.  In fact, this line would read much clearer to me as just a single line (no need for line breaks)